frontends/Makefile.am

```diff
@@ -114,7 +116,8 @@ common_frontend_UNIFIED = \
 	frontends/common/resolveReferences/referenceMap.cpp \
 	frontends/common/resolveReferences/resolveReferences.cpp \
 	frontends/common/parseInput.cpp \
-	frontends/common/constantParsing.cpp
+	frontends/common/constantParsing.cpp \
```

Comment:
we should really keep these files in alphabetical order to reduce merge conflicts.